lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1634,6 +1644,7 @@ instance Arbitrary (ApiTransaction t) where
             <*> genWithdrawals
             <*> pure txStatus
             <*> arbitrary
+            <*> resize 10 (listOf arbitrary)
```

Comment:
> **resize:** _Overrides the size parameter. Returns a generator which uses the given size instead of the runtime-size parameter._

I am pretty sure that this is not what we want here :no_mouth:

Reply:
Why?

Reply:
It'll effectively work because of how `listOf` behaves. I just found it a bit misleading because `resize 10` means that the size parameter will always be `10` whereas QuickCheck typically grow the size parameter to be larger and larger and produce more and more complex cases. `resize 10` circumvent that by forcing the size to always be the same (which acts as a max-size for lists and most other structures, true, but misleading).